config/initializers/shopify_app.rb

```diff
@@ -2,7 +2,7 @@
   api_key: ENV['SHOPIFY_API_KEY'],
   api_secret_key: ENV['SHOPIFY_API_SECRET'],
   host: ENV['HOST'],
-  scope: 'write_products,read_products',
+  scope: ENV.fetch('SCOPES', 'write_products').gsub(/\n/, ''),
```

Comment:
Please, provide SCOPES ENV to the team.

And do not forget to and them on the Render. @MariaGrechko

Reply:
@fooooxmr Okay, understood!